README.md

```diff
@@ -216,24 +216,10 @@ Crossplane resources use the ProviderConfig named ```default``` if no specific P
 
 ### We are now ready to provision resources:
 
-#### Create Network Fabric
```

Comment:
we should also update the "APIs in this Configuration" section that still lists the Network type

Reply:
I believe it's good now.